config/config.txt

```diff
@@ -320,12 +320,22 @@ ANNOUNCE_ADMIN_LOGOUT
 
 ## Map rotation
 ## You should edit maps.txt to match your configuration when you enable this.
+## THIS IS DEPRECATED! Make this a number in a few months.
 MAPROTATION
 
-## Map voting
-## Allows players to vote for their preffered map
-## When it's set to zero, the map will be randomly picked each round
-ALLOW_MAP_VOTING 1
+## Map rotation method
+## Set to 0 for no map rotation, 1 for pref-based maprotation, 2 for vote-based maprotation, 3 for random.
+MAPROTATION_VOTE_METHOD 2
+
+## Map Rotation result method
+## For automatic map votes, this flag will make the vote use weighted random for decisions
+## If this is commented out, a normal "greatest vote amount".
+MAPROTATION_USE_WEIGHTED
+
+## When should the map rotation vote happened, in seconds (for vote-based map rotation)
+## Set this to -1 if you want it to happen at the end of the round.
+## Will default to 300 seconds (5 minutes).
+MAPROTATION_VOTE_DELAY 300
```

Comment:
The default should be near round end, every other game in existence does map votes near round end.